website/src/components/Changelog/GUI.tsx

```diff
@@ -22,19 +28,24 @@ export default function GUI({ title }: { title: string }) {
               a bug where DNS could stop working when you sign out.
             </Link>
           </li>
+          )}
         </ul>
       </Entry>*/}
       <Entry version="1.1.5" date={new Date("2024-07-08")}>
         <ul className="list-disc space-y-2 pl-4 mb-4">
-          <li className="pl-2">
-            The Linux GUI Client is now{" "}
+          {title === "Linux GUI" && (
+          <p>The Linux GUI Client is now{" "}
```

Comment:
Changed `li` to `p` because I saw in most cases if there's only one list item we just use a paragraph tag

Reply:
Our AI overlord says `<p>` cannot be a direct descendant of `<ul>` :-(

Reply:
I tried putting the bare text but TSX didn't seem to like having a text node directly inside `<ul>` either.
Maybe a span or something? What a weird rule... 

I'll just put the `<li>` back, I'm okay with one-item lists personally

Reply:
sgtm